data/presets/presets.json

```diff
@@ -223,6 +236,19 @@
             },
             "name": "Gondola"
         },
+        "aerialway/goods": {
+            "geometry": [
+                "line"
+            ],
+            "fields": [
+                "aerialway/capacity",
+                "aerialway/duration"
+            ],
+            "tags": {
+                "aerialway": "goods"
+            },
+            "name": "Goods"
```

Comment:
I think 'Goods' does not describes properly this tag. What do you think about to change it to 'Goods Aerialway' or 'Cable Lift for Goods'